config/config.yml.example

```diff
@@ -12,4 +12,7 @@ development:
   ignore_environments: []
   allow_uppercase: true
 
+  # Logging
+  loglevel: debug
```

Comment:
should we change it to info or warn?

Reply:
I was torn here as the typical default for a Development environment is debug. The package-based implementations will be updated to default to `warn`.